pkg/network/usm/monitor_testutil.go

```diff
@@ -70,6 +71,9 @@ func (p *protocolMock) Stop(mgr *manager.Manager) {
 func (p *protocolMock) DumpMaps(*strings.Builder, string, *ebpf.Map) {}
 func (p *protocolMock) GetStats() *protocols.ProtocolStats           { return nil }
 
+// IsBuildModeSupported returns always true, as java tls module is supported by all modes.
+func (*protocolMock) IsBuildModeSupported(buildmode.Type) bool { return true }
```

Comment:
choose a const way to return the true, if you decide via one line function do it for all,  else change it to 
```
func (*protocolMock) IsBuildModeSupported(buildmode.Type) bool {
	return true
}
```

Reply:
this is the "format" of the protocolMock struct